src/components/NotificationPanel/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./styles.css";
+import NotificationBox from "./NotificationBox";
+
+const NotificationPanel = ({ notifications }) => (
+  <div className="overlay">
```

Comment:
wait if there's an overlay div here, that's perfect. This is what I want. But it only makes sense to have a single overlay div, so remove the one from App.jsx.